ckanext/stadtzhtheme/plugin.py

```diff
@@ -280,9 +280,12 @@ def get_resource_descriptions(self, res):
 
         file_format = (res.get('format') or 'data').lower()
         if self.descr_config.get(file_format):
+            link = self.descr_config[file_format]['link']
+            if not link:
+                link = ''
             return [
                 self.descr_config[file_format]['description'],
-                self.descr_config[file_format]['link'],
```

Comment:
You could get rid of setting the link-variable and just set a default like so:
`self.descr_config[file_format].get('link', '')`

Reply:
No I can't. I tried it that way first, but 'link' has a value in the dictionary and come back as None if I do it like you suggested.

Reply:
okay cool, thanks for checking :)